web/src/components/new/TagDetailsDialog.vue

```diff
@@ -19,6 +19,14 @@
             <v-col>{{ tag.UncertainCount }}</v-col>
             <v-col>{{ tag.Referenced ? "Yes" : "No" }}</v-col>
           </v-row>
+          <v-row>
```

Comment:
i played around with this and i think this looks nicer:

          <v-row no-gutters>
            <v-col>{{ tag.MatchingCount }}</v-col>
            <v-col>{{ tag.UncertainCount }}</v-col>
            <v-col>{{ tag.Referenced ? "Yes" : "No" }}</v-col>
          </v-row>
          <v-row><v-col></v-col></v-row>
          <v-row no-gutters>
            <v-col cols="4" class="text-caption">Color:</v-col>
            <v-col cols="8"
              ><v-chip small :color="tag.Color">{{ tag.Color }}</v-chip></v-col
            >
          </v-row>
          <v-row><v-col></v-col></v-row>
          <v-row no-gutters>
            <v-col class="text-caption">Definition:</v-col>
          </v-row>